MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult+Internal.swift

```diff
@@ -67,10 +66,10 @@ extension MSALNativeAuthUserAccountResult {
 
             if let result = result {
                 let delegateDispatcher = CredentialsDelegateDispatcher(delegate: delegate, telemetryUpdate: nil)
-                let accessTokenResult = MSALNativeAuthTokenResult(accessToken: result.accessToken,
-                                                                  scopes: result.scopes,
-                                                                  expiresOn: result.expiresOn)
-                Task { await delegateDispatcher.dispatchAccessTokenRetrieveCompleted(result: accessTokenResult, correlationId: result.correlationId) }
+                self.rawIdToken = result.rawIdToken
+                self.account = result.account
+                Task { await delegateDispatcher.dispatchAccessTokenRetrieveCompleted(result: result.accessTokenResult,
+                                                                                     correlationId: result.correlationId) }
                 return
```

Comment:
At this point the `createRetrieveAccessTokenError` can now be private and we can also update the unit tests for that method.

Reply:
Done